backend/test/factories.js

```diff
@@ -31,13 +33,77 @@ export function fakeShow (opts) {
   opts.entryEnd = opts.entryEnd || faker.date.between('2015-01-03', '2015-01-04')
   opts.judgingStart = opts.judgingStart || faker.date.between('2015-01-05', '2015-01-06')
   opts.judgingEnd = opts.judgingEnd || faker.date.between('2015-01-07', '2015-01-08')
+  opts.moreCopies = opts.moreCopies === undefined ? faker.random.boolean() : opts.moreCopies
+  opts.excludeFromJudging = opts.excludeFromJudging === undefined ? faker.random.boolean() : opts.excludeFromJudging
   return Show.create({
     name: opts.name,
     description: opts.description,
     entryCap: opts.entryCap,
     entryStart: opts.entryStart,
     entryEnd: opts.entryEnd,
     judgingStart: opts.judgingStart,
-    judgingEnd: opts.judgingEnd
+    judgingEnd: opts.judgingEnd,
+    moreCopies: opts.moreCopies,
+    excludeFromJudging: opts.excludeFromJudging
   })
 }
+
+function fakeImage (opts) {
+  opts.path = opts.path || faker.system.commonFileName('.jpg')
+  opts.horizDimInch = opts.horizDimInch || faker.random.number({min: 1, max: 100})
+  opts.vertDimInch = opts.vertDimInch || faker.random.number({min: 1, max: 100})
+  opts.mediaType = opts.mediaType || faker.lorem.word()
+  return Image.create({
+    path: opts.path,
+    horizDimInch: opts.horizDimInch,
+    vertDimInch: opts.vertDimInch,
+    mediaType: opts.mediaType
+  })
+}
+
+function fakeEntry (opts) {
+  opts = opts || {}
+  if (!opts.image && !opts.video && !opts.other) {
+    throw Error('No entry item found')
+  }
+  opts.title = opts.title || faker.lorem.words(3)
+  opts.moreCopies = opts.moreCopies === undefined ? faker.random.boolean() : opts.moreCopies
+  opts.comment = opts.comment || faker.lorem.sentence()
+  opts.forSale = opts.forSale === undefined ? faker.random.boolean() : opts.forSale
+  opts.invited = opts.invited === undefined ? faker.random.boolean() : opts.invited
+  opts.awardWon = opts.awardWon || faker.lorem.words(2)
+  opts.yearLevel = opts.yearLevel === undefined ? faker.lorem.word() : opts.yearLevel
+  opts.academicProgram = opts.academicProgram === undefined ? faker.lorem.word() : opts.academicProgram
+  const showPromise = opts.show ? Promise.resolve(opts.show) : fakeShow()
+  const userPromise = opts.user || opts.group ? Promise.resolve(opts.user) : fakeUser()
+  const entryType = opts.image ? IMAGE_ENTRY : opts.video ? VIDEO_ENTRY : OTHER_ENTRY
+  const entryId = opts.image ? opts.image.id : opts.video ? opts.video.id : opts.other.id
+  return Promise.all([showPromise, userPromise])
+    .then((models) => {
+      const show = models[0]
+      const user = models[1]
+      return Entry.create({
+        showId: show.id,
+        studentUsername: user ? user.username : null,
+        groupId: opts.group ? opts.group.id : null,
+        entryType: entryType,
+        entryId: entryId,
+        title: opts.title,
+        comment: opts.comment,
+        moreCopies: opts.moreCopies,
+        forSale: opts.forSale,
+        awardWon: opts.awardWon,
```

Comment:
awardWon?

Reply:
it's documented in Bill's version of the ERD: iirc, during the gallery there's some various named awards that are issued to individual works, and she documents them in the system. We should ask if she wants that feature still. The field is otherwise optional so... we can leave it in and not use it if needed